chromadb/test/test_api.py

```diff
@@ -676,6 +676,53 @@ def test_where_ne_eq_number(api):
     assert len(items["metadatas"]) == 1
 
 
+def test_where_like_string(api):
+    api.reset()
+    collection = api.create_collection("test_where_like")
+    collection.add(**operator_records)
+    items = collection.get(where={"string_value": {"$like": "%wo%"}})
+    assert len(items["metadatas"]) == 1
+    items = collection.get(where={"string_value": {"$like": "%o%"}})
+    assert len(items["metadatas"]) == 2
+    items = collection.get(where={"float_value": {"$like": "%o%"}})
+    assert len(items["metadatas"]) == 0
+    items = collection.get(where={"string_value": {"$nlike": "%o%"}})
+    assert len(items["metadatas"]) == 0
+
+
+def test_where_nlike_string(api):
+    api.reset()
+    collection = api.create_collection("test_where_like")
+    collection.add(**operator_records)
+    items = collection.get(where={"string_value": {"$nlike": "%wo%"}})
+    assert len(items["metadatas"]) == 1
+    items = collection.get(where={"string_value": {"$nlike": "%o%"}})
+    assert len(items["metadatas"]) == 0
+    items = collection.get(where={"string_value": {"$nlike": "%h%"}})
+    assert len(items["metadatas"]) == 2
+    items = collection.get(where={"float_value": {"$nlike": "%wo%"}})
+    assert len(items["metadatas"]) == 0
+
+
+def test_where_valid_operators_like(api):
+    api.reset()
+    collection = api.create_collection("test_where_valid_like_operators")
+    collection.add(**operator_records)
+
+    with pytest.raises(ValueError):
```

Comment:
add `matches=""` to match for specific error we're expecting, in case the API goes on fritz and non-related error is raised.

Reply:
ok, although it's not like that for the other tests.